Examples/Tests/restart/PICMI_inputs_runtime_component_analyze.py

```diff
@@ -97,6 +97,7 @@
     if arg.startswith("amr.restart"):
         restart_file_name = arg.split("=")[1]
         sim.amr_restart = restart_file_name
+        sys.argv.remove(arg)
```

Comment:
@KZhu-ME can you comment why this change was needed? :)
Seems unrelated.

Reply:
This CI test errored out because with the message that there was an unrecognized command line argument `amr.restart`. I also encountered this error locally, and have ran into this issue in the past when trying to add arguments not recognized by the warpx parser. Not sure why it only started failing now for this PR though...